.alexignore

```diff
@@ -10,8 +10,6 @@ website/blog/
 # Once you fix all of the errors, commit all of the changes including this file.
 #
 # To be considerate to the reviewer, please submit each version in different PRs.
-website/versioned_docs/version-0.5/
-website/versioned_docs/version-0.6/
```

Comment:
I think removing this may have been a mistake, and doing so is causing the tests to fail

Don't worry about merge conflicts with master, I'll fix them 👍

Reply:
Sorry, removing 0.6 may have been an mistake - 0.5 should be removed

Reply:
Oh yes, removing 0.6 was definitely a mistake. I'll fix!